hermes-mock/src/main/java/pl/allegro/tech/hermes/mock/HermesMockHelper.java

```diff
@@ -0,0 +1,65 @@
+package pl.allegro.tech.hermes.mock;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.github.tomakehurst.wiremock.WireMockServer;
+import com.github.tomakehurst.wiremock.matching.RequestPatternBuilder;
+import com.github.tomakehurst.wiremock.client.MappingBuilder;
+import com.github.tomakehurst.wiremock.verification.LoggedRequest;
+import org.apache.avro.AvroRuntimeException;
+import org.apache.avro.Schema;
+import tech.allegro.schema.json2avro.converter.JsonAvroConverter;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.UUID;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.aResponse;
+import static com.github.tomakehurst.wiremock.client.WireMock.get;
+import static com.github.tomakehurst.wiremock.client.WireMock.postRequestedFor;
+import static com.github.tomakehurst.wiremock.client.WireMock.urlEqualTo;
+import static com.github.tomakehurst.wiremock.client.WireMock.urlPathMatching;
+
+class HermesMockHelper {
```

Comment:
Not a fan of a bag-of-stuff helper class, but I'm not gonna demand changing it, because it's a private API, so we can modify it later if we need to.